### ISSUE TITLE
Recommended update banner is not displayed if user is not signed in to VPN client

### ISSUE THREAD

**Issue by ValentinaPC** [mozilla-mobile/mozilla-vpn-client#1428]
#### Affected versions:
* 2.4 build; 

#### Affected Platforms:
* Windows 10;

#### Prerequisites:
* Have the VPN app installed;
* Have a FxA account with active subscription for Mozilla VPN;
* Use the Web Socket (http://localhost:8766/) to trigger the recommended update;

#### Steps to reproduce:
1. Open the VPN app;
1. Load http://localhost:8766/ and force the recommended update;
1. Observe the VPN screen;

#### Expected result:
* A blue "update now" banner is displayed.

#### Actual result:
* The banner is not displayed.

#### Notes:
* The banner is displayed after sign in.
* The required update screen is displayed even if the user is not logged in.
* Attaching logs;
[mozillavpn-2021-7-27.txt](https://github.com/mozilla-mobile/mozilla-vpn-client/files/6885211/mozillavpn-2021-7-27.txt)

┆Issue is synchronized with this [Jira Bug](https://mozilla-hub.atlassian.net/browse/VPN-808)
┆author: Data Sync

**Comment by strseb** [comment]
@ValentinaPC - Did we ever show the "update now" banner on the init screen? I can't seem to find a refrence to that. 
As far as i know we only showed "update required" but not the optional ones.

**Comment by jess-cook03** [comment]
I can confirm that is not happening for MacOS and iOS --- I was not prompted (already signed in and opened my VPN app window) and I was not prompted at all to update the app via the banner.
I went to our site and redownloaded and installed the app to update to v2.5 ---
Once I opened this new 2.5 version I was then not prompted with the whats new feature either.
I also did not get any system notifications to indicate updates or whats new either.
